src/Notifier.ts

```diff
@@ -516,13 +516,17 @@ class NotifierClass {
      * Some events require special handling such as showing in-app toasts
      */
     private performCustomEventHandling(ev: MatrixEvent): void {
-        if (ElementCall.CALL_EVENT_TYPE.names.includes(ev.getType()) && SettingsStore.getValue("feature_group_calls")) {
+        if (
+            EventType.CallNotify === ev.getType() &&
+            SettingsStore.getValue("feature_group_calls") &&
+            (ev?.getAge() ?? 0) < 10000
+        ) {
             ToastStore.sharedInstance().addOrReplaceToast({
-                key: getIncomingCallToastKey(ev.getStateKey()!),
+                key: getIncomingCallToastKey(ev.getContent()?.call_id ?? "", ev.getRoomId() ?? ""),
```

Comment:
Since `call_id` is from untrusted event contents, we should guard this with a check that it is actually a string